.github/workflows/build.yml.in

```diff
@@ -146,7 +143,7 @@ jobs:
         with:
           linters: gcc
           run: |
-            bash -o pipefail -c "env LEAN_ABORT_ON_PANIC=1 lake build --wfail -KCI"
+            bash -o pipefail -c "lake exe mk_all || echo "There are unaccounted files, but I am not failing yet!"; env LEAN_ABORT_ON_PANIC=1 lake build --wfail -KCI"
```

Comment:
Can you please add a comment, explaining how this works: something like your explanation should do perfectly.